dev/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/astronomer/astro-runtime:5.0.6-base
+
+USER root
+RUN apt-get update -y && apt-get install -y git
+RUN apt-get install -y --no-install-recommends \
+        build-essential \
+        libsasl2-2 \
+        libsasl2-dev \
+        libsasl2-modules
+ENV SETUPTOOLS_USE_DISTUTILS=stdlib
+
+COPY ../  ${AIRFLOW_HOME}/astro_sdk/
+
+RUN pip install -e "${AIRFLOW_HOME}/astro_sdk/"
```

Comment:
Does it make sense to use the docker multistage here i.e. only copying the installed pip package?

Reply:
this would reduce the final image size but it would also increase the build time a little bit given that this Dockefile would be used for local dag testing purposes only so I'm sure we shall introduce multistage or not here.